.github/pull_request_template.md

```diff
@@ -4,41 +4,39 @@ Thank you for contributing to TiDB-CDC! Please read MD's [CONTRIBUTING](https://
 
 ### What problem does this PR solve? <!--add issue link with summary if exists-->
 
-
 ### What is changed and how it works?
 
-
 ### Check List <!--REMOVE the items that are not applicable-->
 
 Tests <!-- At least one of them must be included. -->
 
- - Unit test
- - Integration test
- - Manual test (add detailed scripts or steps below)
- - No code
+* [ ] Unit test
```

Comment:
This will show up as a todo task on the pr list page, but it's really just a checklist item, which I don't think is appropriate for use here.

Reply:
woo, got this https://jemurai.com/2019/08/15/github-pr-checklist/ i will close this pr